README.rst

```diff
@@ -26,46 +37,21 @@ Why use dual quaternions?
 * easy normalization. Homogeneous tranformation matrices are orthogonal and due to floating point errors operations on them often result in matrices that need to be renormalized. This can be done using the Gram-Schmidt method but that is a slow algorithm. Quaternion normalization is very fast.
 * mathematically pleasing
 
-NOTE: there is no concept of 'from' and 'to' as frame names aren't tracked or used (e.g. use of Pose iso PoseStamped).
-It is up to the user to keep track of those.
-
-Installation & Requirements
----------------------------
-
-In this directory::
-
-    pip install .
-
-Requirements
-~~~~~~~~~~~~
-
-* numpy-quaternion
-* scipy
-* geometry_msgs from ROS
-
-Usage
------
+Installation
+------------
 
-Import using::
+.. code-block:: bash
```

Comment:
will this only work in a certain environment? The requirements went away.

Reply:
No, since the package is hosted on PyPi and the requirements are in `setup.py`, pip install will pull in any requirements that haven't been installed yet.

Reply:
ok, all of the depends work on windows?

Reply:
Just tested on Windows 10. I had to install `c++14` first to make `numpy-quaternion` work, but once that was set it did succeed. From some issues on the repo it seems like [windows isn't a priority for the dev](https://github.com/moble/quaternion/issues/75) though. The output error code clearly stated that the error was the lack of `C++14` and had a link to a download.